src/components/layout/header/Menu/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+import classNames from 'classnames/bind';
+
+import styles from './Menu.module.scss';
+
+const cx = classNames.bind(styles);
+
+export const Menu = () => {
+  const gameTypes = ['LEAGUE OF LEGENDS', 'BATTLEGROUNDS', 'OVERWATCH 2', 'MINCRAFT'];
```

Comment:
객체 파일로 따로 관리해 주시면 어떨까요?
추후 게임이 더 추가될 수도 있고 유지보수 시 nav 파일로 관리하면 좋지 않을까 생각합니다!
```suggestion
  const NAV_LIST = [
  { id: 0, gameType: 'LEAGUE OF LEGENDS' },
  { id: 1, gameType: 'BATTLEGROUNDS' },
  { id: 2, gameType: 'OVERWATCH 2' },
  { id: 3, gameType: 'MINCRAFT' },
];
```

Reply:
따로 관리하기 위해 constants에 gameList 파일을 따로 만들었고 객체보다는 배열이 적절한 거 같아서 구조는 바꾸지 않았습니다!
`export const GAME_LIST = ['LEAGUE OF LEGENDS', 'BATTLEGROUNDS', 'OVERWATCH 2', 'MINCRAFT'];
`

Reply:
객체로 넣을 경우 id 값이 들어 있기 때문에 map으로 렌더링 시 지금처럼 index를 넣지 않고 객체의 id 값으로 부여할 수 있지 않을까요?

Reply:
id를 0123으로 하고 그 값을 부여하는 것과 index를 이용하는 것이 크게 차이가 없다고 생각해서 기존의 배열 형태를 유지했습니다!